storybook/src/stories/Global/TopBar.stories.tsx

```diff
@@ -119,6 +121,29 @@ export const _TopBar = () => {
   ])
   const notificationsHistory = object("Notifications History", allNotifications);
 
+  const userConfig = object("User Notes", {
```

Comment:
Please change this variable and its json data as per comments given in storybook/src/stories/Global/GlobalUI.stories.tsx

Reply:
Done